standards/accessibility.md

```diff
@@ -0,0 +1,30 @@
+# Accessibility Standards
+
+The Library provides resources that should be accessible to all patrons. When building front-end applications that patrons interact with, we MUST build the apps with accessibility in mind. This is a general overview and the Design Toolkit's [Developer Checklist](https://nypl.github.io/design-toolkit/resources/development-checklist.html) MUST be followed.
+
+### Implementation
+
+Applications MUST be fully keyboard navigable.
+
+Applications MUST be built as progressive web apps.
+
+Applications MUST support working with Javascript turned off. This includes making navigation and form submissions work as intended. Enhancements to the applications SHOULD be built afterwards.
+
+Developers SHOULD have a working knowledge of the basics of HTML and CSS. This will go a long way to building applications the correct way from the beginning. This alone covers writing valid HTML, creating valid forms with valid `label`s and `input`s, creating page landmarks, creating valid `img` elements, and so on.
+
+Developers SHOULD build user interfaces (UIs) using the native HTML elements intended for their behavior. For example, a dropdown SHOULD be built using the native `<select>` and `<option>` elements instead of a `<button>` with a list `<ul>` dropdown. Unless it is absolutely necessary, UIs behavior can be built with other HTML, but they MUST work as intended in different browsers.
+
+Developers MUST build apps for users with slow connections in mind. This means having the 'above the fold' CSS and HTML be delivered quickly.
+
+Developers SHOULD be components that provide valid HTML over the framework's implementation. For example, when building a definition list `dl`, the only valid HTML allowed as children are `dt` and `dd`. React cannot return a component in the form of
+
+    <dt>Term</dt>
+    <dd>Definition</dd>
+
+and require the developer to wrap those elements in a `div` or `span`. Since this creates invalid HTML, it is preferable to not create a smaller component for that HTML and instead allow it to be part of a bigger component. It is better to write a big and accessible React component than smaller and inaccessible React components.
+
+### Tools
+
+Developers MUST run their apps through web accessibility tools.
```

Comment:
A reference of NYPL tools would be ideal. I think Ricardo tackled that in a Sprint way back in time. Maybe it is listed in the design toolkit?